settings.php

```diff
@@ -36,4 +36,15 @@
             get_string('setting:showseblinks', 'quizaccess_seb'),
             get_string('setting:showseblinks_desc', 'quizaccess_seb'),
             $links, $links));
+
+    $settings->add(new admin_setting_configtext('quizaccess_seb/downloadlink',
+            get_string('setting:downloadlink', 'quizaccess_seb'),
+            get_string('setting:downloadlink_desc', 'quizaccess_seb'),
+            'https://safeexambrowser.org/download_en.html',
+            PARAM_URL));
+
+    $settings->add(new admin_setting_configcheckbox('quizaccess_seb/quizpasswordrequired',
+        get_string('setting:quizpasswordrequired', 'quizaccess_seb'),
```

Comment:
Looks like formatting for this section of setting is different to the rest of the file. Would be good to make it consistent.